### PR TITLE
Add a Procfile so we can run as a govuk::app in puppet

### PR DESCRIPTION
**Pull request by tuzz**
https://trello.com/c/HSre7K10/453-style-the-mvp-email-collection-page-needed-by-jan

The #publishing-frontend team want to be able to work on the gem without
having to mount it into another Rails application first. We can use the
dummy Rails app the test suite uses to achieve this.
  
Other PR: https://github.com/alphagov/govuk-puppet/pull/7014

### CODE REVIEW

File: spec/component_guide/component_index_spec.rb [hunk 1]
```diff
@@ -3,13 +3,6 @@
 describe 'Component guide index' do
   # Load ordering test can only fail if run as the first test in suite
   # https://github.com/rails/rails/issues/12168
-  it 'renders using gem layout not app layout after viewing a page on the application' do
```
[the remark this answers is not in the document]
Comment (tuzz) [reply spec/component_guide/component_index_spec.rb]: Yeah, having looked again at the test, I agree. I've removed that commit.

**Comment by fofr** [comment]
Is this also compatible with running on Heroku for review apps? That would be 🎆

**Comment by nickcolley** [comment]
We believe so, but will make any updates if needed to make that happen.

**Comment by fofr** [comment]
Aww
screen shot 2018-01-05 at 15 33 52

**Comment by nickcolley** [comment]
I wonder if it's possible to tell heroku not to look at the top level directories?

**Pull request merged**